### DISCUSSION TITLE
Navigating pages on jamulus.io

### DISCUSSION THREAD

**Discussion by gene96817** [jamulussoftware/jamulus#2089] [Documentation]
Since I am stuck with my planned installation of 3.8.1, I am poking around jamulus.io.
Nice to find that jamulus.io/wiki gives a longer list of pages than the navigation box on the right.
It would be very helpful if each page had a last-changed date and the matching jamulus version number. It could be in the bottom (left or right) corner.

Where are the pages documenting the earlier releases? 
Some pages were created prior to 3.8.1. It would be nice to know which releases these pages are useful. I can only assume the page is not accurate for 3.8.1.

**Comment by ignotus666** [comment]
To the best of my knowledge, all pages are relevant to 3.8.1 - but if you (or anyone else) detects that there is outdated information, we will be glad to hear about it to make the relevant corrections. The fact that a page was created or updated before the latest release doesn't mean its information is no longer applicable, though. If you want to see when a file was last edited, you can easily do that by checking the website's github repository.

There was a proposal to maintain a repository with older versions of the website but I can't remember where that thread went...

**Reply by gene96817** [reply]
I'll come back with a list of suggestions once I have 3.8.1 installed.
